mars/dataframe/expressions/arithmetic/core.py

```diff
@@ -547,22 +547,43 @@ def _tile_both_dataframes(cls, op):
                                      chunks=out_chunks, dtypes=df.dtypes,
                                      index_value=df.index_value, columns_value=df.columns)
 
+    @classmethod
+    def _tile_scalar(cls, op):
+        new_op = op.copy()
+        left = op.inputs[0]
+
+        df = op.outputs[0]
+        out_chunks = []
+        for left_chunk in left.chunks:
+            out_op = op.copy().reset_key()
+            out_chunk = out_op.new_chunk([left_chunk], shape=left_chunk.shape, index=left_chunk.index,
+                                         index_value=left_chunk.index_value,
+                                         columns_value=left_chunk.columns)
+
+            out_chunks.append(out_chunk)
+
+        return new_op.new_dataframes(op.inputs, df.shape, nsplits=left.nsplits, dtypes=df.dtypes,
+                                     index_value=df.index_value, columns_value=df.columns, chunks=out_chunks)
+
     @classmethod
     def tile(cls, op):
         if all(isinstance(inp, DATAFRAME_TYPE) for inp in op.inputs):
             return cls._tile_both_dataframes(op)
-
+        if isinstance(op.inputs[0], DATAFRAME_TYPE) and np.isscalar(op.inputs[1]):
+            return cls._tile_scalar(op)
         raise NotImplementedError
 
     @classproperty
     def _operator(self):
         raise NotImplementedError
 
     @classmethod
-    def _calc_properties(cls, x1, x2):
```

Comment:
When would `x2` be None?

Reply:
When it's called from `_new_chunks`. Because in `_tile_scalar`, I only create the chunk of dataframe (not include scalar). Is that right?